semver.md

```diff
@@ -206,9 +206,9 @@ to the users of your software. Once these intentions are clear, flexible (but
 not too flexible) dependency specifications can finally be made.
 
 A simple example will demonstrate how Semantic Versioning can make dependency
-hell a thing of the past. Consider a library called "Firetruck." It requires a
-Semantically Versioned package named "Ladder." At the time that Firetruck is
-created, Ladder is at version 3.1.0. Since Firetruck uses some functionality
+hell a thing of the past. Consider a library called "Firetruck", that requires a
```

Comment:
This first comma becomes unnecessary when you restructure the sentence like this because "that requires" modifies "Ladder" and isn't an independent clause. I'd restructure this as:

> Consider a library called "Firetruck" that requires a Semantically Versioned package named "Ladder". At the time that...

I think that flows better.

Reply:
Ok, I'll submit another PR shortly.

Reply:
Might I recommend this:
> Consider a library called "Firetruck" that requires a Semantically Versioned package named "Ladder"; at the time that Firetruck is created Ladder is at version 3.1.0.